src/network/networkprotocol.h

```diff
@@ -937,6 +938,8 @@ enum AccessDeniedCode {
 	SERVER_ACCESSDENIED_ALREADY_CONNECTED,
 	SERVER_ACCESSDENIED_SERVER_FAIL,
 	SERVER_ACCESSDENIED_CUSTOM_STRING,
+	SERVER_ACCESSDENIED_SHUTDOWN,
+	SERVER_ACCESSDENIED_CRASH,
```

Comment:
I just looked at SERVER_FAIL and all of its uses were related to the password not matching up.  Sounds to me like SERVER_ACCESSDENIED_SERVER_FAIL would need to get renamed as well.